mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 
 msgid "spyder_flower_sandy_notweesher"
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
+
+msgid "spyder_wayfarer1_bismuth2"
+msgstr "¡Gorrón!"
+
+msgid "spyder_route2_roddick2"
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
+
+msgid "menu_down_key"
+msgstr "Tecla Bajar"
+
+msgid "menu_secondary_select_key"
+msgstr "Tecla Cancelar"
+
+msgid "spyder_route4_wulf2"
+msgstr "Ooh, ¡me siento tan castrado!"
+
+msgid "menu_left_key"
+msgstr "Tecla Izquierda"
+
+msgid "dark_robo"
+msgstr "Robo Oscuro"
```

Comment:
What is the context? Is this a robot? Because "robo" is "theft" in Spanish, so maybe we should use a different word.

Reply:
```suggestion
msgstr "Robot Oscuro"
```